### DISCUSSION TITLE
Versions priority

### DISCUSSION THREAD

**Discussion by niprobin** [itsToggle/plex_debrid#487] [Q&A]
Hey there, I am not sure I found the information in the README or in the Wiki but how versions are prioritised? My idea would be to have 1 version to check for 4k and if 0 results check 2nd version for very specific 1080p releases.

My understanding is that it start with the 1st version listed at Options/settings/scraper settings/Versions, do you confirm?

I love the work here, amazing stuff!

**Comment by Kozikodi** [comment]
Have you figured this out?

**Reply by jagvirdehal** [reply]
If I'm not wrong there's a setting for minimum retries, so if the other versions fail X many times then that version will be used.
